6.5: Uploaded rt patch

MichaelDeets opened the pull request:
Based on [v6.5-rc7-rt4](https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-rt-devel.git/tag/?h=v6.5-rc7-rt4).
Removed patches related to ARM.
Removed patches already applied by EEVDF and presumably TT, otherwise will fail during patching.
Must be applied after EEVDF/TT, hence the naming convention being `0002` and not `0001`

ventureoo commented:
Looks good for me, build was also fine. But note that it would be nice if you included in your patch a commit (https://github.com/CachyOS/linux/commit/ddab46a8132224a5a114a9da1a29c204262ea7d5) to disable the CONFIG_EXPERT dependency, as it greatly simplifies config maintenance for us.

ptr1337 commented:
Please add the CONFIG_EXPERT removal and rename the patch to "0001-rt.patch" so that it will fit to our PKGBUILDs.
If it does work with EEVDF thats great. Then we can use for the RT kernel also EEVDF.

MichaelDeets commented:
I've removed CONFIG_EXPERT requirement and renamed the patch.

To work with EEVDF I simply removed a few conflicting patch entries. One could apply the unmodified RT.patch and change EEVDF to work with it, but I found this method easier.

ventureoo commented on 6.5/misc/0001-rt.patch:
```diff
@@ -0,0 +1,7029 @@
+ config PREEMPT_RT
+ 	bool "Fully Preemptible Kernel (Real-Time)"
+-	depends on EXPERT && ARCH_SUPPORTS_RT
++	depends ARCH_SUPPORTS_RT
```
```suggestion
+	depends on ARCH_SUPPORTS_RT
```
You missed the ``on`` statement
